pages/v/[id]/plyr.tsx

```diff
@@ -0,0 +1,38 @@
+import { GetStaticProps, GetStaticPaths } from 'next';
+
+import PlayerPage from '../../../components/player-page';
+import { getPropsFromPlaybackId, Props } from '../../../lib/player-page-utils';
+
+type Params = {
+  id: string;
+};
+
+export const getStaticProps: GetStaticProps = async (context) => {
+  const { params } = context;
+  const { id: playbackId } = params as Params;
+  const props = await getPropsFromPlaybackId(playbackId);
+
+  return { props };
+};
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  return {
+    paths: [],
+    fallback: true,
+  };
+};
+
+const PlyrPage: React.FC<Props> = ({ playbackId, videoExists, shareUrl, poster, aspectRatio }) => {
```

Comment:
non-blocking: This inversion feels a little off to me. We abstract out a `plyr` page, and then have a shared `PlayerPage`, but that then expects to be given a `playerType`. We can probably clean this up a bit more, though it's a low priority imo.

Reply:
Fair point. I think the issues stems from the root level having to be a "page" or a route in Next.js

- `/pages/v/[id]/plyr` << this is plyr, and we need 1 file for this
- `/pages/v/[id]` << this is a generic page, we need 1 where we can a/b test different players
- `/pages/v/[id]/mux-player` << this is plyr, and we need 1 file for this

All 3 of these are "PlayerPages" with the same `PlayerPage` layout, which is generic.

Maybe a better refactor would be

- `/pages/v/[id]`
- `/pages/v/[id]/[playerType]`

That abstraction might feel a bit better, will try that in the `mux-player` followup and see how it feels

Reply:
addressed in: https://github.com/muxinc/stream.new/pull/94